<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassificationResultPanel</class>
 <widget class="QWidget" name="ClassificationResultPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="nnName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="imgName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="prevImgButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>562</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextNNButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>560</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Next NN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevNNButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>560</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous NN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextImgButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>560</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="saveResultButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>600</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Result</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
